html/elements/iframe.json

```diff
@@ -1008,7 +1008,7 @@
                 "notes": "Currently only available on macOS High Sierra 10.13.4 beta, and in Safari Technology Preview 47+."
               },
               "safari_ios": {
-                "version_added": "11.1",
+                "version_added": "11.3",
                 "notes": "Currently only available on iOS 11.3 beta."
```

Comment:
Sorry for the drive-by change request, but can we remove this note and the Safari one? This kind of note doesn't age well. MDN authors: please don't add things like this.

Reply:
Removed!